### ISSUE TITLE
Pap_Stats_Trans_SalesStatsProcessorTask inserted 2 records instead of 1

### ISSUE THREAD

**Issue by lnikodem** [QualityUnit/pap-public#38]
check Pap_Stats_Trans_SalesStatsProcessorTask, if is not used cron, it is possible that two immediate requests on update for some day (hour) insert same record into db and then there is this record multiple times

ZRU-FTDZS-055

**Comment by jurajsim** [comment]
a nemal by tento kod na zaciatku tomu zabranit?
```php
 private function processTransactionsForDay($dateInserted) {
        $transStatsDay = new Pap_Db_TransStatsDay();
        $transStatsDay->setDateInserted($dateInserted);
        $transStatsDay->delete();

        $this->fillTransactionsStatsAff($dateInserted);
        $this->fillTransactionsStats($dateInserted);
}
```
tym padom tam nemozu vzniknut 2 rovnake dni nie?

**Comment by lnikodem** [comment]
fixed in 5.4.1.2
